src/api-doc/api-doc.yaml

```diff
@@ -3420,45 +3467,69 @@ paths:
               properties:
                 name:
                   type: string
+                  description: Name of the question
                 question:
                   type: string
+                  description: 'What question you want to add '
                 type:
                   type: string
+                  description: Give the type of the question like feedback or rating
                 options:
                   type: 'null'
                 no_of_stars:
                   type: number
+                  description: Number of stars to be set
                 status:
                   type: string
+                  description: Status of the question
                 category:
-                  type: 'null'
-                rendering_data:
                   type: object
                   properties:
-                    value:
-                      type: string
-                    class:
-                      type: string
-                    disabled:
-                      type: boolean
-                    noOfstars:
-                      type: string
-                    position:
+                    evaluating:
                       type: string
-                    validation:
+                      description: For mentor category evaluating as mentor
+                  description: 'default category should be as null '
+                rendering_data:
+                  type: object
+                  properties:
+                    validators:
                       type: object
                       properties:
                         required:
                           type: boolean
-                meta:
-                  type: 'null'
+                          description: 'Setting this question is mandatory or not '
```

Comment:
@sumanvpacewisdom description for "rendering_data" Setting this question is mandatory or not ? is this correct ?